packages/drivers/local-driver/src/localDocumentServiceFactory.ts

```diff
@@ -76,6 +76,7 @@ export class LocalDocumentServiceFactory implements IDocumentServiceFactory {
             defaultHash,
             resolvedUrl.endpoints.ordererUrl ?? "",
             resolvedUrl.endpoints.storageUrl ?? "",
+            "",
```

Comment:
This seems like an error?

Reply:
Yeah my package versions seem to be a bit off so I'm looking into fixing these errors. I will remove this!